dm/syncer/streamer_controller.go

```diff
@@ -285,21 +285,15 @@ func (c *StreamerController) GetEvent(tctx *tcontext.Context) (event *replicatio
 	switch ev := event.Event.(type) {
 	case *replication.RotateEvent:
 		// if is local binlog, binlog's name contain uuid information, need to save it
-		// if is remote binlog, need to add uuid information in binlog's name
-		c.Lock()
-		containUUID := c.setUUIDIfExists(string(ev.NextLogName))
-		uuidSuffix := c.uuidSuffix
-		c.Unlock()
-
-		if !containUUID {
```

Comment:
we may switch 2 remote when using relay and meet error, so we need to add suffix if there's ever set `c.uuidSuffix`.

the comments before should be:
```
// if is local binlog but switch to remote on error, need to add uuid information in binlog's name
```

the lock can be removed though, only `getEvent` routine is using it.